dashboard/app/models/user_level.rb

```diff
@@ -110,17 +110,11 @@ def submitted_or_resubmitted?
   def after_submit
     submitted_level = Level.cache_find(level_id)
 
-    # Create peer reviews after submitting a peer_reviewable solution
+    # Some levels that belong to the professional learning program are peer reviewable.
+    # When submitting such a level, we create a record for a pending peer review, so that
+    # it becomes available for another teacher in the program to review.
```

Comment:
Is there ever a case where a user is "auditing" a Professional Learning course, but isn't actually enrolled?

(Do we create Peer Reviews in that case?)

Reply:
I don't actually know.  It looks like you can access PL scripts by direct link without being enrolled in the course, but in other places we block access if you're not enrolled (and we certainly don't provide any friendly UI for browsing to a PL course without being enrolled in it).

Let me query for unenrolled activity on these courses and get back to you.

Reply:
Hit a wall trying to run that query; asking for help in Slack.